src/gridworld/GridWorld.cc

```diff
@@ -251,7 +251,11 @@ void GridWorld::add_agents(GroupHandle group, int n, const char *method,
                 }
 
                 agent->set_dir(turn_mode ? (Direction) pos_dir[i] : NORTH);
+#if defined(_WIN32)                
+                agent->set_pos({pos_x[i], pos_y[i]});
```

Comment:
Is there really not one way that works for both platforms? I suspect there should be some standards compliant version of this that works everywhere.

Reply:
I think the WIN32 define way is the correct structure initialization. This does also works on Linux. Also farther down in the file you use another call to set pos which also works on both. So working is either 
```
                agent->set_pos({pos_x[i], pos_y[i]});   // Ok 
```
or
```
                agent->set_pos(Position{pos_x[i], pos_y[i]}); // Ok
```
but not 
```
               agent->set_pos((Position) {pos_x[i], pos_y[i]}); // failed
```
I would have removed the old call. But I did not want to change your original code, since something like this would introduce a bad error and I don't know whether you have tests to pick this up.

Reply:
I think it's safe to remove the old call in favour of either the first of second one.